specified the currently supported auth method (#5)
NETWAYS/icingaweb2-module-servicenow-import: doc/03-Configuration.md

Since we wanted to make it as customizable as possible, we don't have a hard-coded API method. Therefore, you have to provide the full endpoint for the table here (e.g. `api/now/table/cmdb_ci_server`).

**ServiceNow API Username:** User to authenticate against the ServiceNow API.

> Currently only Basic Auth is supported, meaning you would use the same credentials as you do to login into the UI.

Reviewer: ```suggestion
Currently only Basic Auth is supported to authenticate against ServiceNow.
```
